spark/src/main/kotlin/com/adevinta/spark/components/menu/DropdownMenu.kt

```diff
@@ -148,81 +154,152 @@ public fun DropdownMenuItem(
     leadingIcon: @Composable (() -> Unit)? = null,
     trailingIcon: @Composable (() -> Unit)? = null,
     enabled: Boolean = true,
-    colors: MenuItemColors = MenuDefaults.itemColors(),
-    contentPadding: PaddingValues = MenuDefaults.DropdownMenuItemContentPadding,
+    contentPadding: PaddingValues = PaddingValues(
+        horizontal = 16.dp,
+        vertical = 8.dp,
+    ),
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
 ) {
-    MaterialDropdownMenuItem(
-        text = text,
-        onClick = onClick,
-        modifier = modifier,
-        leadingIcon = leadingIcon,
-        trailingIcon = trailingIcon,
-        enabled = enabled,
-        colors = colors,
-        contentPadding = contentPadding,
-        interactionSource = interactionSource,
-    )
+    val colors = MenuDefaults.itemColors()
+    Row(
+        modifier = modifier
+            .clickable(
+                enabled = enabled,
+                onClick = onClick,
+                interactionSource = interactionSource,
+                indication = rememberRipple(true),
+            )
+            .fillMaxWidth()
+            // Preferred min and max width used during the intrinsic measurement.
+            .sizeIn(
+                minWidth = 112.dp,
+                maxWidth = 280.dp,
+                minHeight = 48.dp,
```

Comment:
Better put magic numbers as constants

Reply:
No need here since they're relevant only to this component in this place and they're associated to semantics arguments so we can avoid repeating ourselves and adding complexity.